pkg/slices/slices.go

```diff
@@ -444,13 +444,13 @@ func ChunkFunc[S ~[]E, E any](slice S, size int, callback func(chunk S)) {
 // Except returns a new slice for all items that are not found in the slice of
 // items to exclude.
 func Except[S ~[]E, E comparable](slice S, exclude S) S {
-	set := sets.NewFromSlice(exclude)
+	set := maps.NewSetFromSlice(exclude)
 	return ExceptSet(slice, set)
 }
 
 // ExceptSet returns a new slice for all items that are not found in the set of
 // items to exclude.
-func ExceptSet[S ~[]E, E comparable](slice S, exclude sets.Set[E]) S {
+func ExceptSet[S ~[]E, E comparable](slice S, exclude maps.Set[E]) S {
```

Comment:
Should be able to use `sets.Set` here right?

Reply:
Yup 👍🏻